src/Components/Filtering/Multiselect.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Delete this disable, if you have this error and your code works then you should be able to delete whichever var is flagging it!